src/types/TodoStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoStatus {
```

Comment:
It is a filter type. Use a more descriptive name.

```suggestion
export enum FilterType / FilterBy {
```